PluginFramework/Interfaces/PluginTypes/IPluginOverhead.cs

```diff
@@ -1,7 +1,7 @@
-﻿using PluginFramework.src.DataContainer;
-using System.Windows.Forms;
+﻿using PluginFramework.DataContainer;
+//using System.Windows.Forms;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/simonaberle/XmlFormatter/pullRequest?prid=5634024)